docs/sparkr.md

```diff
@@ -262,6 +262,67 @@ head(df)
 {% endhighlight %}
 </div>
 
+### Applying User-defined Function
+
```

Comment:
It will be good to add an introduction here that there are two kinds of user-defined functions we support in SparkR. Something like 
```
In SparkR we support two kinds for user-defined functions
1. Run a given function on a large dataset using dapply. 
2. Run many functions in parallel using spark.lapply. 
```